src/main/java/com/opencart/datamodel/LoginModel.java

```diff
@@ -1,31 +1,38 @@
 package com.opencart.datamodel;
 
-public class LoginModel {
 
-    private final String email = "dfghszdfgh@gmail.com";
-    private final String newEmail = "newEmail@gmail.com";
-    private final String password = "dfghszdfgh";
-    private final String newPassword = "newPassword";
-    private final String newFirstName = "NewFirstName";
+public class LoginModel {
 
+    private String email = "dfghszdfgh@gmail.com";
+    private String password = "dfghszdfgh";
```

Comment:
Could you, please, move it to LoginRepositry too?

Reply:
Done